src/proposals/20250306/SparkEthereum_20250306.t.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { Base }     from 'spark-address-registry/Base.sol';
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { IMetaMorpho, IMorpho, MarketAllocation, MarketParams } from 'metamorpho/interfaces/IMetaMorpho.sol';
+
+import { ReserveConfig } from "src/test-harness/ProtocolV3TestBase.sol";
+
+import 'src/test-harness/SparkTestBase.sol';
+
+import { ChainIdUtils } from 'src/libraries/ChainId.sol';
+
+contract SparkEthereum_20250306Test is SparkEthereumTests {
+
+    address internal constant AGGOR_BTCUSD_ORACLE = 0x4219aA1A99f3fe90C2ACB97fCbc1204f6485B537;
+    address internal constant CBBTC_USDC_ORACLE   = 0x663BECd10daE6C4A3Dcd89F1d76c1174199639B9;
+
+    constructor() {
+        id = '20250306';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 21930000);  // Feb 26, 2024
```

Comment:
Do we need it if we call `setupDomains()`?

Reply:
Yeah good point can remove

Reply:
Removed